terraform/stacks/main/domains_broker.tf

```diff
@@ -203,6 +203,34 @@ output "domains_broker_listener_arns" {
 }
 
 /* new broker ALB */
+
+resource "aws_iam_user" "domain_broker_v2" {
+  name = "domain_broker_v2"
+}
+
+resource "aws_iam_access_key" "domain_broker_v2_access_key" {
+  user = "${aws_iam_user.domain_broker_v2.name}"
+}
+
+resource "aws_iam_user_policy" "domain_broker_v2_policy" {
+  name = "domain_broker_v2_policy"
+  user = "${aws_iam_user.domain_broker_v2.name}"
+  policy = <<EOF
+  {
+    "Version": "2015-12-01",
+    "Statement": [
+      {
+        "Action": [
+          "elasticloadbalancing:*"
+        ],
+        "Effect": "Allow",
+        "Resource": "*"
```

Comment:
this seems overly-broad. Can we narrow it down, at least to the names of the alb's it's expected to manage?

Reply:
Yes, I am very on board with that. And of course I think I just found it. Bleh, I couldn't figure it out originally, lol.